### PR TITLE
Add .create_gantt entry, associated media and new tags doc

### LINKED ISSUE #4563: [Term Entry] Python Plotly .create_gantt()
### Reason/inspiration (optional)

We would like a new term entry in the `Data Visualization` concept for Plotly: `.create_gantt()`. The entry should go in a new file under `docs/content/plotly/concepts/figure-factory/terms/create-gantt/create-gantt.md`

The entry should describe the term and how it is used. Also include:

- An introductory paragraph that describes the concept
- A 'Syntax' section that includes the syntax for the concept (if applicable)
- An 'Example' section that shows the concept in use

Please refer to the [entry templates](https://github.com/Codecademy/docs/blob/main/documentation/), [content standards](https://github.com/Codecademy/docs/blob/main/documentation/content-standards.md), and [markdown style guide](https://curriculum-documentation.codecademy.com/content-guidelines/markdown-style-guide/) when working on the PR for this issue.

### Entry Type (select all that apply)

New Term for Existing Concept Entry

### Code of Conduct

- [X] By submitting this issue, I agree to follow Codecademy Doc's [Code of Conduct](https://github.com/Codecademy/docs/blob/main/.github/CODE_OF_CONDUCT.md).

### For Maintainers

- [X] Labels added
- [x] Issue is assigned

### PR DESCRIPTION
**Pull request by rgo444**
### Description

Added a new term entry for Plotly: **.create_gantt()**, along with an associated media image of the plot output. I also updated the tags document to include a new tag: Plotly :round_pushpin:

### Issue Solved

Closes #4563

### Type of Change

- Adding a new entry

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x] I have linked any issues that are relevant to this PR in the `Issues Solved` section.

### CODE REVIEW

**Review by mamtawardhani — CHANGES_REQUESTED** [review]
Hey @rgo444! Thanks for contributing at Codecademy!

Please do the following:
(1) Sign the CLA
(2) Could you please rename the file to `create_gantt.md`, instead of `create-gantt.md`
(3) Please remove the `tags.md` file from this branch.
(4) I've mentioned some suggestions for the entry, could you please review and modify them at your earliest convenience?

Thank you!

File: content/plotly/concepts/figure_factory/terms/create_gantt/create-gantt.md [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+  - 'Data Science
+  - 'Data Visualization'
+Tags:
+  - 'Graphs'
+  - 'Libraries'
+  - 'Plotly'
+CatalogContent: # Please use course/path landing page slugs, rather than linking to individual content items. If listing multiple items, please put the most relevant one first
+  - 'learn-python-3'
+  - 'paths/data-science'
+---
```
[the remark this answers is not in the document]
Comment (rgo444) [reply content/plotly/concepts/figure_factory/terms/create_gantt/create-gantt.md:14]: Hi @mamtawardhani !

I have completed the requested changes, hope I managed to update the pull request correctly :dizzy_face:

Thank you!

File: content/plotly/concepts/figure_factory/terms/create_gantt/create_gantt.md [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+CatalogContent: 
+  - 'learn-python-3'
+  - 'paths/computer-science'
+---
+
+The **`.create_gantt`** method in the `plotly.figure_factory` module is a function used in `Plotly` to generate Gantt charts. Gantt charts are graphical representations of project schedules that illustrate tasks, their durations, and dependencies. This method allows users to create interactive Gantt charts within their Plotly figures, providing a visual overview of project timelines and progress.
+
+## Syntax
+
+"The syntax for the .create_gantt() method in Plotly is customizable. Below is a general structure of it: "
```
Comment (mamtawardhani) [inline content/plotly/concepts/figure_factory/terms/create_gantt/create_gantt.md:21]:
```suggestion
The syntax for the .create_gantt() method in Plotly is customizable. Below is a general structure of it: 
```

File: content/plotly/concepts/figure_factory/terms/create_gantt/create_gantt.md [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+
+```pseudo
+
+plotly.figure_factory.create_gantt(df, colors=None, index_col=None, show_colorbar=False, reverse_colors=False, title=’Gantt Chart’, ....)
+
+```
+
+`.create_gantt` takes the following arguments:
```
Comment (mamtawardhani) [inline content/plotly/concepts/figure_factory/terms/create_gantt/create_gantt.md:29]:
```suggestion
The `.create_gantt()` method takes the following arguments:
```

File: content/plotly/concepts/figure_factory/terms/create_gantt/create_gantt.md [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+
+- `df`: This is the only required parameter for this method, a dataframe or a list that serves as input data for the chart.
+- `colors`: Either a plotly scale name, list, dictionary or a tuple.
+- `index_col`: The column header to be used as indexing column.
+- `show_colorbar`: Boolean, determines if the colorbar will be visible.
+- `reverse_colors`: Boolean, serves as a reverser of selected colors.
+- `title`: Sets the title of the chart.
```
Comment (mamtawardhani) [inline content/plotly/concepts/figure_factory/terms/create_gantt/create_gantt.md:36]:
```suggestion
- `df`: Required input data, either a `data frame` or a list, serving as the foundation for the Gantt chart's content.
- `colors`: Specifies the color scheme used in the chart, allowing customization through options like a Plotly scale name, a `list` of colors, a `dictionary` mapping values to colors, or a `tuple` defining the color range.
- `index_col`: Identifies the column header to be used as the indexing column in the Gantt chart, aiding in the organization of tasks or data points along the `y-axis`.
- `show_colorbar`: A `boolean` parameter determining the visibility of the color bar in the Gantt chart.
- `reverse_colors`: A `boolean` parameter enabling the reversal of selected colors in the chart.
- `title`: Sets a descriptive title for the Gantt chart.
```

File: content/plotly/concepts/figure_factory/terms/create_gantt/create_gantt.md [hunk 5]
```diff
@@ -0,0 +1,57 @@
...
+
+**Note:** The ellipsis (...) indicates that there can be additional optional arguments beyond those listed here.
```
Comment (mamtawardhani) [inline content/plotly/concepts/figure_factory/terms/create_gantt/create_gantt.md:38]:
```suggestion
> **Note:** The ellipsis (...) indicates that there can be additional optional arguments beyond those listed here.
```

File: content/plotly/concepts/figure_factory/terms/create_gantt/create_gantt.md [hunk 6]
```diff
@@ -0,0 +1,57 @@
...
+
+df = [dict(Task="Task A", Start='2024-01-01', Finish='2024-02-02'), 
+	dict(Task="Task B", Start='2024-03-01', Finish='2024-11-11'), 
+	dict(Task="Task C", Start='2024-08-06', Finish='2024-09-21')] 
+
+fig = ff.create_gantt(df) 
+fig.show()
+
+```
+**Output:**
```
Comment (mamtawardhani) [inline content/plotly/concepts/figure_factory/terms/create_gantt/create_gantt.md:55]:
```suggestion

The code above produces the following output:
```

**Review by mamtawardhani — CHANGES_REQUESTED** [review]
Hey @rgo444! I've suggested some changes in the entry, could you please review those and modify them at your earliest convenience? 
Also, you can remove the older `create-gantt.md` file from this branch, keep only the new `create_gantt.md` file.

Thanks!

**Comment by mamtawardhani** [comment]
Hey @rgo444! 
Here's a suggestion, I see that you have added a lot of files for this PR. Could you close this PR and create a new one with the suggestions you modified? That will be easy actually.

And sorry for the inconvenience caused, please make sure the path is `docs/content/plotly/concepts/figure-factory/terms/create-gantt/create-gantt.md`, so the `create-gantt/create-gantt.md` folder and file name should be fine.

Thank you!

**Comment by mamtawardhani** [comment]
Yes @rgo444, it would be `create-gantt.md`.
Just make sure you apply the suggestions that I had given! Thanks! :)

**Comment by rgo444** [comment]
Done deal @mamtawardhani! Have an excellent weekstart!

**Pull request closed without merging**